versionalchemy/utils.py

```diff
@@ -181,13 +185,13 @@ class _JSONEncoded(TypeDecorator):
     def process_bind_param(self, value, dialect):
         if value is None:
             return None
-        elif isinstance(value, basestring):
+        elif isinstance(value, six.string_types):
             value = json.loads(value)
 
         if self.json_type is not None and not isinstance(value, self.json_type):
             raise ValueError('value of type {} is not {}'.format(type(value), self.json_type))
 
-        return json.dumps(value, ensure_ascii=False, encoding='utf8', cls=VAJSONEncoder)
+        return json.dumps(value, ensure_ascii=False, cls=VAJSONEncoder)
```

Comment:
`encoding='utf-8'` was already the default for `simplejson`, and it's _not_ a supported kwarg for `json.dumps`